src/amadeus/namespaces/booking.js

```diff
@@ -0,0 +1,25 @@
+import HotelBookings from './booking/hotel_bookings';
+
+/**
+ * A namespaced client for the
+ * `/v1/booking` endpoints
+ *
+ * Access via the {@link Amadeus} object
+ *
+ * ```js
+ * let amadeus = new Amadeus();
+ * amadeus.booking;
+ * ```
+ *
+ * @param {Client} client
+ * @property {HotelBookings} hotelBookings
+ * @protected
+ */
+class Booking {
+  constructor(client) {
+    this.client    = client;
+    this.hotelBookings = new HotelBookings(client);
+  }
+}
+
+export default Booking;
```

Comment:
should the newlines at the end of the files be added? I am not sure about JS.

Reply:
Nope, we don't keep extra lines in the end. Also, none of the previous files has it.